client/components/deposits-information/loading.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardHeader, CardBody } from '@wordpress/components';
+import Gridicon from 'gridicons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Loadable from 'components/loadable';
+
+const DepositsInformationLoading = (): any => {
```

Comment:
I think we should avoid `any` at all costs. Can we change this to `JSX.Element`? Or a `React.Node` — I think that's the React element type, not sure though.

Reply:
Yes, `JSX.Elements` did the job, it was an oversight.